neuralop/losses/data_losses.py

```diff
@@ -402,6 +401,77 @@ def __call__(self, y_pred, y, h=None, **kwargs):
             normalization constant for reduction, by default None
         """
         return self.rel(y_pred, y, h=h)
+
+class PointwiseQuantileLoss(object):
+    def __init__(self, alpha, reduce_dims = 0, reductions='sum'):
+        """PointwiseQuantileLoss computes Quantile Loss described in [1]
+
+        Parameters
+        ----------
+        alpha : float
+            value, between 0 and 1, of the proportion of points
+            in the output domain expected to fall within predicted quantiles
+        reduce_dims : int, optional
+            dimensions to reduce when summing, by default 0
+            This loss was formulated for functions with a co-domain in R, 
+            so for now only 0 is supported
+        reductions : str, optional
+            how to apply reduction (sum or mean), by default 'sum'
+
+        References
+        1. Calibrated Uncertainty Quantification for Operator Learning via Conformal Prediction
+            Ma, Z., Azizzadenesheli, K., Anandkumar, A., 2024. https://arxiv.org/html/2402.01960v1
+        """
+        
+        super().__init__()
+
+        self.alpha = alpha
+
+        if isinstance(reduce_dims, int):
+            self.reduce_dims = [reduce_dims]
+        else:
+            self.reduce_dims = reduce_dims
+
+        if self.reduce_dims is not None:
+            if isinstance(reductions, str):
+                assert reductions == 'sum' or reductions == 'mean'
```

Comment:
Add an error message. I think assert can be used as a function now, which may make it slightly clearer.